### DISCUSSION TITLE
Renovate does not identify Debian Bookworm as an update

### DISCUSSION THREAD

**Discussion by aarongoldenthal** [renovatebot/renovate#22921] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

Renovate identifies Debian container image updates up to `bullseye`, but does not identify `bookworm` as an update.

Reproduction at https://github.com/aarongoldenthal/renovate-debian-bookworm.

See related discussion [here](https://github.com/renovatebot/renovate/discussions/22836#discussioncomment-6233648).

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by aarongoldenthal** [comment]
FYI - Bookworm just showed up in the last half hour (at least in my [self hosted example](https://gitlab.com/aarongoldenthal/renovate-track-image-os/-/merge_requests/25), still not on [GitHub](https://github.com/aarongoldenthal/renovate-debian-bookworm)).

**Reply by viceice** [reply]
yes, there was an update of the release data a few days ago

**Reply by HonkingGoose** [reply]
## PR that updated Debian release data

> yes, there was an update of the release data a few days ago

See PR #23263 for the details on that.

## Why did it take so long for us to detect the LTS?

Quote from @rarkins: [^rarkins-comment]

> We should work out why it took us a month to detect this LTS. Did they take a month to update their site?

@rarkins did you, or your team, find out _why_ it took us a month? There is no issue or PR for this yet.

## What is the source of truth?

Quote from @JamieMagee about the source files: [^source-files]

> Actually, with a bit more digging, this is the source for `debian-info-data` and `ubuntu-info-data`:
> 
> * https://debian.pages.debian.net/distro-info-data/debian.csv
> * https://debian.pages.debian.net/distro-info-data/ubuntu.csv

[^source-files]: https://github.com/renovatebot/renovate/pull/14881#issuecomment-1086751248 
[^rarkins-comment]: https://github.com/renovatebot/renovate/pull/23263#discussion_r1257820214

**Reply by viceice** [reply]
delay was probably due to a broken workflow since the removal of the azure tasks stuff
